lib/github.js

```diff
@@ -85,53 +86,89 @@ function aliasGet (msg, done) {
 }
 
 function queryGithub (msg, done) {
-  var cache = opts.cache
+  var cache  = opts.cache
 
   github.authenticate({
-    type: 'token',
+    type : 'token',
     token: opts.token
   })
 
-  var params = {
-    user: msg.user,
-    repo: msg.repo
-  }
-
-  github.repos.get(params, (err, repo) => {
-    if (err) return done(err)
-    if (!repo) return done()
+  async.parallel({
+    getRepository  : function (callbackFn) {
```

Comment:
callback functions are usually written as `cb` or `done` - best to follow the convention of existing code in the repo. Also goes for putting spaces after var name, e.g. 'foo   : bar', go with whatever the existing code uses (this stuff should be caught by a linter)